lib/apiservers/engine/backends/backends.go

```diff
@@ -256,7 +257,8 @@ func InsecureRegistries() []string {
 
 // syncContainerCache runs once at startup to populate the container cache
 func syncContainerCache() error {
-	log.Debugf("Updating container cache")
+	op := trace.NewOperation(context.Background(), "")
```

Comment:
Taking into account that this is a relatively common case when empty parameters are passed to the NewOperation, I would consider having a new constructor such as NewEmptyOperation that will do exactly same thing.